src/pudl/metadata/fields.py

```diff
@@ -2215,6 +2215,26 @@
         "type": "boolean",
         "description": "indicates if the boiler is a heat recovery steam generator (HRSG).",
     },
+    "in_rate_base": {
+        "type": "boolean",
+        "description": (
+            "Whether or not a record from the detailed FERC1 accounting tables should "
+            "be considered allowable in a utility's rate base based on utility "
+            "accounting standards. "
+            "This flag was mannually compiled by RMI utility accounting experts "
+            "based on the xbrl_factoid and sometimes varies based on the utility_type "
+            "plant_status or plant_function as well."
+        ),
+    },
+    "in_revenue_requirement_review": {
+        "type": "boolean",
+        "description": (
+            "Whether or not a record from the detailed income statement data should be "
+            "considered in a utility's revenue requirement. This flag was mannually "
```

Comment:
nit: the word "considered" could be a little confusing here. What do you mean by this?

Reply:
"is typically included" ?